extensions/cdc-debezium/src/main/java/com/hazelcast/jet/cdc/impl/ChangeRecordCdcSourceP.java

```diff
@@ -54,7 +54,8 @@ public ChangeRecordCdcSourceP(
     @Nullable
     @Override
     protected ChangeRecord map(SourceRecord record) {
-        if (record == null) {
+        if (record == null || record.topic().startsWith("__debezium")) {
+            // internal Debezium messages about e.g. Heartbeat uses such topics
             return null;
```

Comment:
Another alternative could be 
```java
Struct sourceRecordValue = (Struct) sourceRecord.value();
String sourceSchemaName = sourceRecordValue.schema().name();
return sourceSchemaName.startsWith ("io.debezium")
```

Reply:
True, it's the same as with value IMHO, so I left only scanning value. But good point :)